app/app/urls.py

```diff
@@ -355,7 +355,7 @@
     re_path(r'^settings/(.*)?', marketing.views.email_settings, name='settings'),
 
     # marketing views
-    url(r'^leaderboard/(.*)', marketing.views.leaderboard, name='leaderboard'),
+    url(r'^leaderboard/(\w*)\/?(\w*)', marketing.views.leaderboard, name='leaderboard'),
```

Comment:
I'm not a huge fan of including the sorting option in the uri.  Can you update this to be an optional `GET` param?